test/e2e/tools_bucket_web_test.go

```diff
@@ -105,7 +105,7 @@ func TestToolsBucketWebExternalPrefix(t *testing.T) {
 func TestToolsBucketWebExternalPrefixAndRoutePrefix(t *testing.T) {
 	t.Parallel()
 
-	e, err := e2e.NewDockerEnvironment("route-prefix")
+	e, err := e2e.NewDockerEnvironment("route-prefix-extrtprf")
```

Comment:
We get ` unexpected error: name can't be smaller than 1 or over 16 character long due to docker network name constraints, got: route-prefix-extrtprf` in the test.

Maybe this could work?
```suggestion
	e, err := e2e.NewDockerEnvironment("rt-prfx-xtrtprf")
```